test/plugins/ruby.plugin.bats

```diff
@@ -18,17 +18,21 @@ function local_teardown {
 }
 
 @test "plugins ruby: remove_gem is defined" {
+  load ../../plugins/available/ruby.plugin
```

Comment:
isnt this duplicated? why do we need to load again?

Reply:
Fixed!